resources/content_transform.go

```diff
@@ -47,14 +39,30 @@ func (h *MapEndpointHandler) mapContent(w http.ResponseWriter, r *http.Request,
 		writeError(w, err, transactionID, uuid, r.RequestURI)
 		return
 	}
+	if contentHasBeenDeleted(upPlaceholder) {
+		writeMessageForDeletedContent(w, transactionID, uuid, r.RequestURI)
+		return
+	}
 
 	w.Header().Add("Content-Type", "application/json")
 	encoder := json.NewEncoder(w)
 	encoder.Encode(upPlaceholder)
 	log.WithField("transaction_id", transactionID).WithField("uuid", uuid).WithField("request_uri", r.RequestURI).Info("Transformation successful")
 }
 
-func writeError(w http.ResponseWriter, err error, transactionID string, uuid string, requestURI string) {
+func writeError(w http.ResponseWriter, err error, transactionID, uuid, requestURI string) {
 	log.WithField("transaction_id", transactionID).WithField("uuid", uuid).WithField("request_uri", requestURI).WithError(err).Error(fmt.Sprintf("Returned HTTP status: %v", http.StatusUnprocessableEntity))
 	http.Error(w, err.Error(), http.StatusUnprocessableEntity)
 }
+
+func writeMessageForDeletedContent(w http.ResponseWriter, transactionID, uuid, requestURI string) {
+	msg := fmt.Sprintf("Content has been deleted. transaction_id=\"%v\" uuid=\"%v\" request_uri=\"%v\"", transactionID, uuid, requestURI)
+	log.Info(msg)
+	w.Header().Add("Content-Type", "text/plain")
```

Comment:
This should be json, and `{"message":"Delete event"}` to be consistent with other mappers.